### PR TITLE
Media Item API

### PR DESCRIPTION
**Pull request by egemahlec**
Please review the modifications and let me know your comments. 
As far as I tested, it is working correctly.

### CODE REVIEW

File: API Project/src/main/java/com/bounswe/models/CulturalHeritage.java [hunk 1]
```diff
@@ -93,69 +93,6 @@ public void setCity(String city) {
...
-        this.createdTime = createdTime;
-    }
-
-    public Date getUpdatedTime() {
-        return updatedTime;
-    }
-
-    public void setUpdatedTime(Date updatedTime) {
-        this.updatedTime = updatedTime;
-    }
```
Comment (YmirKhang) [inline API Project/src/main/java/com/bounswe/models/CulturalHeritage.java:159]: Nice work but why did you remove all of the getters and setters. We need at least the getters and setters for the string fields.

**Review by YmirKhang — CHANGES_REQUESTED** [review]
Getters and setters for the string fields?

File: API Project/src/main/java/com/bounswe/models/CulturalHeritage.java (hunk 1 above)
Comment (yigitozkavci) [reply API Project/src/main/java/com/bounswe/models/CulturalHeritage.java:159]: They were duplicated, apparently @emir350z

**Review by yigitozkavci — APPROVED** [review]
Seems fine to me, nice job indeed :)

**Review by YmirKhang — APPROVED** [review]

**Pull request merged**